### DISCUSSION TITLE
Issue when Handling  Chat Exports above 2GB

### DISCUSSION THREAD

**Discussion by sumitshrestha** [Pustur/whatsapp-chat-parser-website#41] [General]
Hey,

I’ve tried rendering various chat exports of different sizes. When the file size is below 2 GB, it renders perfectly. However, files larger than 2 GB result in a blank page on Firefox or nothing at all on Chrome. Is this a known issue or a deliberate limitation due to memory considerations? Is there a way to override this through settings?

**Comment by Pustur** [comment]
@sumitshrestha I don't even have a 2GB chat to test this, but at that point I assume that you're asking too much from the browser

**Reply by sumitshrestha** [reply]
i have 32GB RAM in my PC. i don't think it should be problem. 

At this point, i am wondering if we could have a offline processing where it could parse the file and put it in db like SQLite or MySQL where could load 100 chat based on how user is scrolling.

**Reply by Pustur** [reply]
I'm gonna be honest, I think this is an issue that affects such a small % of people that I'm not going to bother with it.
The simplest workaround would be to unzip the archive, remove the biggest files from there and compress the files again.

**Reply by Pustur** [reply]
Ok fine, I actually looked into it. It's a limitation of JSZip. 
Could you test using this url to see if it still crashes? https://develop--whatsapp-chat-parser.netlify.app/

Also, I opened #42 if you want to continue there
